config/locales/gd/organisations.yml

```diff
@@ -484,5 +484,7 @@ gd:
     view_less: féach níos lú
     what_we_do: Cad atá á dhéanamh ag %{title}
     works_with_statement:
+      two:
+      few:
```

Comment:
Looks like these crept into the wrong commit. Should they have been in the `Add missing pluralisation keys` instead? There are several examples dotted around this commit.